config/deploy.rb

```diff
@@ -9,7 +9,7 @@
 set :keep_releases, 10
 
 set :rbenv_type, :user
-set :rbenv_ruby, '2.1.5'
+set :rbenv_ruby, '2.2.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.